extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1099,9 +1099,19 @@ export class AmpStoryPage extends AMP.BaseElement {
       // No-op.
       return Promise.resolve();
     } else {
-      return mediaPool.register(
-        /** @type {!./media-pool.DomElementDef} */ (mediaEl)
-      );
+      const parentEl = mediaEl.parentElement;
+      let layoutPromise = Promise.resolve();
+      if (
+        parentEl.tagName === 'AMP-VIDEO' ||
```

Comment:
Not super familiar with this code, but why do we have to check for the `parentEl`? At this point in the codepath, is it guaranteed that we always get the `<src>` elements before the `<amp-video>/<amp-audio>` elements? Could there be a scenario where `<amp-video>` has no children yet?

Reply:
> Could there be a scenario where <amp-video> has no children yet?

When there's a `background-audio` we just create an `audio` tag without using `amp-audio`, and the parent is, I think, `amp-story-page` (If you're curious, there's a bit more context here: https://github.com/ampproject/amphtml/pull/27782#issuecomment-614348695)